sjb/config/test_cases/test_branch_origin_extended_conformance_install_update.yml

```diff
@@ -83,6 +83,12 @@ extensions:
         sed -i 's|go/src|data/src|' _output/local/releases/rpms/origin-local-release.repo
         sudo cp _output/local/releases/rpms/origin-local-release.repo /etc/yum.repos.d/
     - type: "script"
+      title: "build service-catalog user-broker image"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        REGISTRY=openshift/ NO_DOCKER=1 make -C cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog user-broker
```

Comment:
Something to be mindful of (maybe it doesn't matter?) is that this is using the upstream build process rather than the origin tooling (build-go.sh).

Reply:
yeah we talked about this last night -- this doesn't need to be versioned etc so should be ok